TestProjects/HDRP_HybridTests/Assets/SampleScenes/BRGGameObjects/RenderBRG.cs

```diff
@@ -332,12 +340,27 @@ public JobHandle OnPerformCulling(BatchRendererGroup rendererGroup, BatchCulling
         return jobHandleOutput;
     }
 
+    public static Material LoadPickingMaterial()
+    {
+        Shader shader = Shader.Find("Hidden/HDRP/BRGPicking");
+        Material material = new Material(shader);
+
+        // Prevent Material unloading when switching scene
+        material.hideFlags = HideFlags.HideAndDontSave;
+
+        return material;
+    }
+
+    Material m_pickingFallbackMaterial;
```

Comment:
Maybe this shouldn't have "fallback" in the name anymore?

Reply:
Fixed